src/app/components/card/Card.vue

```diff
@@ -12,8 +12,10 @@
 export interface Props {
   backgroundColor?: string
 }
+
 withDefaults(defineProps<Props>(), {
-  backgroundColor: 'bg-background-brighten dark:bg-background-darken',
+  backgroundColor:
+    'bg-background-card-bright border border-[0.1px] border-accent-line shadow-[var(--color-shadow-card)] dark:bg-background-darken',
```

Comment:
1. Can there be a border of less than a pixel? 👀
2. Does `border-accent-line` exist?
3. Setting the border color seems necessary now